### PR TITLE
Add remote attestation setup and how to use instruction

### PR DESCRIPTION
**Pull request by bktan8**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,29 +220,43 @@ cd tdx/guest-tools
 TD_IMG=<path_to_td_qcow2_image> ./run_td_virsh.sh
 ```
 
-## Verify TD Guest
+<a id="verify-td-guest"></a>
+## 7. Verify TD Guest
+
+1. Confirm a TD guest is running.
```
Comment (hector-cao) [inline README.md:226]: This is not a very accurate method of verification, user may have an other instance of running QEMU that is not related to TD. I dont think we need to give users this instructions, if users cannot ssh into the guest, that means the guest is not running,

File: README.md [hunk 2]
```diff
@@ -192,29 +220,43 @@ cd tdx/guest-tools
...
+
+An example output:
 
-1. Log into the guest.
+```
+username        85046       1  0 Mar12 ?        00:33:46 qemu-system-x86_64 -D /tmp/tdx-guest-td.log -accel kvm -m 2G -smp 64 -name td,process=td,debug-threads=on -cpu host -object tdx-guest,id=tdx -machine q35,hpet=off,kernel_irqchip=split,memory-encryption=tdx,memory-backend=ram1 -object memory-backend-ram,id=ram1,size=2G,private=on -bios /usr/share/ovmf/OVMF.fd -nographic -daemonize -nodefaults -device virtio-net-pci,netdev=nic0_td -netdev user,id=nic0_td,hostfwd=tcp::10022-:22 -drive file=/home/sdp/tdx/guest-tools/image/vmware-tdx-guest-ubuntu-23.10.qcow2,if=none,id=virtio-disk0 -device virtio-blk-pci,drive=virtio-disk0 -device vhost-vsock-pci,guest-cid=3 -pidfile /tmp/tdx-demo-td-pid.pid
+```
 
+2. Log into the guest.
+v
```
Comment (hector-cao) [inline README.md:239]: Is "v" a typo ?

**Review by hector-cao — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -45,10 +65,11 @@ git clone https://github.com/canonical/tdx.git
 ```
 
 3. Run the script. <br>
+
 Note: If you're behind a proxy, use `sudo -E` to preserve user environment.
 
 ```bash
-cd tdx
+cd ~/tdx
```
[the remark this answers is not in the document]
Comment (bktan8) [reply README.md:72]: removed.

File: README.md (hunk 1 above)
Comment (bktan8) [reply README.md:226]: ok.  removed.

File: README.md (hunk 2 above)
Comment (bktan8) [reply README.md:239]: typo.  fixed.

**Review by hector-cao — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -227,22 +269,222 @@ Example output:
...
+Failed to extend rtmr[2]
+[../tdx_attest.c:537] Inappropriate ioctl for device
+Failed to extend rtmr[3]
+```
+
+Note: You can ignore the `Failed to extend rtmr` messages.
+
+You should also find a `quote.dat` file generated.
+
+2. Next step is to attest with the [Intel Trust Authority]((https://www.intel.com/content/www/us/en/security/trust-authority.html) service.  For this, you will need to subscribe and obtain an API key.
```
[the remark this answers is not in the document]
Comment (bktan8) [reply README.md]: fixed.

**Review by hector-cao — APPROVED** [review]
LGTM
